share/picongpu/examples/FoilLCT/include/picongpu/param/fieldAbsorber.param

```diff
@@ -35,7 +35,7 @@ namespace picongpu
         namespace absorber
         {
             // This variable is not required, defined for convenience
-            constexpr uint32_t THICKNESS = 64;
+            constexpr uint32_t THICKNESS = 16;
```

Comment:
Why do you changed the default thickness too?

Reply:
For the same reason. 64 indicates the example wanted a very strong absorption, so I put 16 which is very strong for PML